app/controllers/instances_controller.rb

```diff
@@ -21,6 +25,15 @@ def create # Create the game instance
     end
   end
 
+  def join
+    @instance = Instance.find_by_pin(params[:pin])
+    if @instance.blank?
+      redirect_to root_path, notice: "Lobby not found"
+    else
+      redirect_to instance_path(@instance)
+    end
```

Comment:
If an instance is found then we should also create a player, player must be create with the matching instance_id.